test/libsolidity/SemanticTest.cpp

```diff
@@ -218,27 +58,27 @@ bool SemanticTest::run(ostream& _stream, string const& _linePrefix, bool const _
 	for (auto& test: m_tests)
 	{
 		bytes output = callContractFunctionWithValueNoEncoding(
-			test.call.signature,
-			test.call.value,
-			test.call.arguments.rawBytes()
+			test.call().signature,
+			test.call().value,
+			test.call().arguments.rawBytes()
 		);
 
-		if ((m_transactionSuccessful == test.call.expectations.failure) || (output != test.call.expectations.rawBytes()))
+		if ((m_transactionSuccessful == test.call().expectations.failure) || (output != test.call().expectations.rawBytes()))
 			success = false;
 
-		test.failure = !m_transactionSuccessful;
-		test.rawBytes = std::move(output);
+		test.setFailure(!m_transactionSuccessful);
+		test.setRawBytes(std::move(output));
 	}
 
 	if (!success)
 	{
 		AnsiColorized(_stream, _formatted, {BOLD, CYAN}) << _linePrefix << "Expected result:" << endl;
 		for (auto const& test: m_tests)
-			_stream << formatFunctionCallTest(test, _linePrefix, false, true & _formatted);
+			_stream << test.format(_linePrefix, false, true & _formatted) << endl;
```

Comment:
This is the newline fix. Problem was, that `format(...)` added it already which resulted in a redundant newline for single-line function call tests.

Reply:
What is the idea behind `true & _formatted`?

Reply:
This determines whether the output should be colorized using `AnsiColorized` or not. If this test case runs in `isoltest` (`_formatted` is true) we want the output to be colorized. If it runs in `soltest` (`_formatted` is false) we do not want to have that behavior because it would break the CI output in case of a expectation / result mismatch. 

But I think it should just be `_formatted` instead of `true & _formatted` ;)